finalProject/cipher.js

```diff
@@ -0,0 +1,159 @@
+
+
+
+function encrypt(message, encryption){
+  var alphabet = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z','A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z'];
+  var cipher = message.toUpperCase().split("");
+  var final = []
+  for ( var i = 0; i <= cipher.length; i++){
+    if(cipher[i] == alphabet[0]){
```

Comment:
I believe that you could accomplish this statement using index of. And that would omit the need for all of your else ifs